hello.txt

```diff
@@ -0,0 +1 @@
+hello world
```

Comment:
you shouldn't write like this. Use camel naming convention

Reply:
bug fixed